resource/interface/client/progress.lua

```diff
@@ -220,3 +204,50 @@ RegisterCommand('cancelprogress', function()
 end)
 
 RegisterKeyMapping('cancelprogress', 'Cancel current progress bar', 'keyboard', 'x')
+
+local function deleteProgressProps(playerProps)
+    if not playerProps then return end
+    for i = 1, #playerProps do
+        local prop = playerProps[i]
+        if DoesEntityExist(prop) then
+            DeleteEntity(prop)
+        end
+    end
+    playerProps = nil
```

Comment:
This does basically nothing. You should be assigning `createdProps[id] = nil` rather than the variable referencing it.